lib/src/widget.dart

```diff
@@ -138,6 +138,7 @@ class _AdvancedDrawerState extends State<AdvancedDrawer>
                 textDirection:
                     widget.rtlOpening ? TextDirection.rtl : TextDirection.ltr,
                 child: ScaleTransition(
+                  alignment: Alignment.centerLeft,
```

Comment:
It looks like it should be based on a widget.rtlOpening` value, isn't?

Reply:
yes